libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -610,6 +610,16 @@ AP_GPS::first_unconfigured_gps(void) const
 }
 
 void
+AP_GPS::broadcast_first_configuration_failure_reason(void) const {
+    uint8_t unconfigured = first_unconfigured_gps();
+    if(drivers[unconfigured] == NULL) {
+        GCS_MAVLINK::send_statustext_all(MAV_SEVERITY_INFO, "GPS %d: was not found", unconfigured + 1);
```

Comment:
The idea is that we only get here if we are looking for a GPS device that hasn't been detected. That's considered a failure, we specified hardware we can't find.